apps/interpolate_generator/interpolate_generator.cpp

```diff
@@ -122,7 +122,7 @@ class Interpolate : public Halide::Generator<Interpolate> {
                 }
                 normalize
                     .reorder(c, x, y)
-                    .bound(c, 0, 3)
+                    .bound(c, 0, 4)
```

Comment:
I think interpolate is 4 input channels and 3 output channels, based on its filter.cpp

Reply:
Nice catch.

Reply:
This also implies the bound(c, 0, 4) calls above should be reverted to bound(c, 0, 3), because I think those are both the output Func.